### PR TITLE
Fixes #6974 but also adds hella epik references

### LINKED ISSUE #6974: I'm fairly sure Poly's parroting is broken
I would stand next to him and keep yelling shit, he would never speak anything except his default lines.

### PR DESCRIPTION
**Pull request by 9600bauds**
I am so witty

Performs the auto-close on and thus fixes #6974

### CODE REVIEW

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 1]
```diff
@@ -118,8 +119,29 @@
 		held_item.loc = src.loc
 		held_item = null
 	walk(src,0)
+	desc = pick("It's just resting.", \
```
Comment (ghost) [inline code/modules/mob/living/simple_animal/parrot.dm]: I love you.

Comment (ghost) [inline code/modules/mob/living/simple_animal/parrot.dm]: Can you make this change every time it's examined?

**Comment by clusterfack** [comment]
Approved

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 2]
```diff
@@ -113,6 +114,30 @@
...
+			"It's pining for the fjords.", \
+			"It just prefers kippin' on it's back.", \
+			"That parrot is definitely deceased.", \
+			"You know a dead parrot when you see one, and you're looking at one right now.", \
+			"It's dead, that's what's wrong with it.", \
+			"It's bleeding demised.", \
+			"It's passed on.", \
+			"This parrot is no more.", \
+			"It has ceased to be.", \
+			"It's expired and gone to meet it's maker.", \
```
Comment (Intigracy) [inline code/modules/mob/living/simple_animal/parrot.dm:131]:
```
>it's maker
```

@9600bauds wow wtf

Comment (9600bauds) [inline code/modules/mob/living/simple_animal/parrot.dm:131]: I already fixed that nerd

**Pull request merged**